internal/dev_server/adapters/api.go

```diff
@@ -87,6 +98,14 @@ func (a apiClientApi) getFlags(ctx context.Context, projectKey string, href *str
 	return flags, nil
 }
 
+func (a apiClientApi) getEnvironments(ctx context.Context, projectKey string) ([]ldapi.Environment, error) {
```

Comment:
Probably should page through. You can use the `getFlags` as an example.

Reply:
yep, good call.

Reply:
added.  Also made the paging logic generic.